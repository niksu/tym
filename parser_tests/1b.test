%hello
c() :- a(). a(b).
