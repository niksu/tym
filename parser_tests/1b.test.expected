input contents |%hello
c() :- a(). a(b).
|
(no query given)
stringed file contents (size=57, remaining=1943)
|predicate_c() :- predicate_a().
predicate_a(constant_b).|
stringed file contents strlen=56
